dev/start.sh

```diff
@@ -7,8 +7,8 @@ docker-compose build
 
 if [ ! -f data_key ]; then
 	echo "Generating data key"
+  docker-compose pull
```

Comment:
Mind getting rid of the rest of the tabs in this file before you merge so that the spacing is consistent?